server/parsec/cli/run.py

```diff
@@ -539,6 +628,44 @@ def run_cmd(
 
         jinja_env = get_environment(template_dir)
 
+        match (allowed_client_agent, with_client_web_app):
+            case (AllowedClientAgent() as cooked_allowed_client_agent, _):
+                pass
+            case (None, Path()):
+                cooked_allowed_client_agent = AllowedClientAgent.NATIVE_OR_WEB
+            case (None, None):
+                cooked_allowed_client_agent = AllowedClientAgent.NATIVE_ONLY
+
+        if openbao_server_url is None:
+            openbao_config = None
+        else:
+            if openbao_secret_mount_path is None:
+                raise ValueError(
+                    "--openbao-secret-mount-path is required when --openbao-server-url is provided"
+                )
+
+            auths = []
+            if openbao_auth_hexagone is not None:
+                auths.append(
+                    OpenBaoAuthConfig(
+                        id=OpenBaoAuthType.HEXAGONE,
+                        mount_path=openbao_auth_hexagone,
+                    )
+                )
+            if openbao_auth_pro_connect is not None:
+                auths.append(
+                    OpenBaoAuthConfig(
+                        id=OpenBaoAuthType.PRO_CONNECT,
+                        mount_path=openbao_auth_pro_connect,
+                    )
+                )
+
+            openbao_config = OpenBaoConfig(
+                server_url=openbao_server_url,
+                secret_mount_path=openbao_secret_mount_path,
+                auths=auths,
```

Comment:
What happens if neither hexagone nor pro connect are set up ?

Reply:
Then the GUI won't display any SSO connection options, which is not great for the user, but not an issue from the Parsec server point of view ^^

This is a good point though: I've added a simple check there to return an error if no ` --openbao-auth-*` has been provided :+1: